script/run-nightwatch.sh

```diff
@@ -9,13 +9,28 @@ trap 'kill $(jobs -p)' EXIT
 
 BUILDTYPE=${BUILDTYPE:-development}
 
-# Run the api server and the webserver.
-node src/test-support/mockapi.js &
-node src/test-support/test-server.js --buildtype ${BUILDTYPE} &
+# Check to see if we already have an API server running on port 3000
+if [ `nc -z localhost 3000; echo $?` -ne 0 ]; then
```

Comment:
this is hard coded to port 3000, and the rest of this process can be overwritten by an ENV variable.